test/integration/test.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+'use strict';
+const path = require('path');
+const Listr = require('listr');
+const tempy = require('tempy');
+const execa = require('execa');
+const del = require('del');
+
+const packages = new Map([
+	['got', 'https://github.com/sindresorhus/got'],
+	['ava', 'https://github.com/avajs/ava']
```

Comment:
I don't have that many large repos anymore, but here are some more suggestions:

* https://github.com/kevva/to-ico
* https://github.com/kevva/download
* https://github.com/kevva/brightness
* https://github.com/kevva/decompress
* https://github.com/kevva/npm-conf
* https://github.com/imagemin/imagemin
* https://github.com/chalk/wrap-ansi